MagmaSafe.Borders/Repositories/IServersRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using MagmaSafe.Borders.Entities;
+using System.Threading.Tasks;
+
+namespace MagmaSafe.Borders.Repositories {
+    public interface IServersRepository {
```

Comment:
Nome de interface é no singular.

```suggestion
    public interface IServerRepository 
    {
    }
```